metricbeat/module/haproxy/haproxy.go

```diff
@@ -260,10 +260,13 @@ func (p *unixProto) run(cmd string) (*bytes.Buffer, error) {
 		return response, err
 	}
 
-	_, err = io.Copy(response, conn)
+	recv, err := io.Copy(response, conn)
 	if err != nil {
 		return response, err
```

Comment:
maybe at the mean time improve this error to errors.Wrap(...) to give more context there?

Reply:
Ah, not a bad idea!